easybuild/easyconfigs/p/pubtcrs/pubtcrs-master-intel-2019.01.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'MakeCp'
+
+name = 'pubtcrs'
+version = 'master'
```

Comment:
@sassy-crick Since `master` will/might move over time, we should take a different approach here, i.e. install a specific commit and slap our own version on it (typically a datestamp of the form `20190125` that corresponds with the date of the commit).

https://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#alternative-formats-for-sources is helpful here, but you shouldn't use the `git_config` in this case, but just instruct `eb` to download the tarball corresponding to a specific commit, using https://github.com/phbradley/pubtcrs/archive/f30b96a.tar.gz in this case.

Are you up for making that change?